apps/reflect/src/forms/TwitchBotForm.tsx

```diff
@@ -13,17 +13,23 @@ import Select from "@mui/material/Select";
 import CustomAlert from "../components/CustomAlert";
 import AppSnackbar from "../components/app/AppSnackbar";
 import { SneacbarSeverity } from "../enums";
-import { ITwitchBotFormSubmitData } from "src/types";
+import { ITwitchBotConfig, ITwitchBotFormSubmitData } from "src/types";
 
 const TwitchBotForm = () => {
   const [alertIsOpen, setAlertIsOpen] = useState<boolean>(false);
   const [snackbarIsOpen, setSnackbarIsOpen] = useState<boolean>(false);
   const [buttonEnabled, setButtonEnabled] = useState<boolean>(false);
 
-  const { data: botActivityEnabledConfig, isLoading } =
-    trpc.twitchBot.getConfig.useQuery({
-      configName: "bot_activity_enabled",
-    });
+  const { data: configs, isLoading } = trpc.twitchBot.getAllConfigs.useQuery();
```

Comment:
[for future] Let's find a way to easily move these calls to server, so that we don't have to write ugly loading logic on clientside.